fre/pp/checkoutScript.py

```diff
@@ -20,7 +20,7 @@
 
 #############################################
 
-def _checkoutTemplate(experiment, platform, target):
+def _checkoutTemplate(experiment, platform, target, branch='empty'):
```

Comment:
This looks fine to me (as a not-quite python person), but it might be more pythonic to set the default to be `None`. (We can always improve it later)

def _checkoutTemplate(experiment, platform, target, branch=None):